utilities/unittests/test_pytest_utils.py

```
@@ -893,9 +893,7 @@ def test_get_artifactory_server_url_default_server_empty_dict(self, mock_get_sec
         mock_session.config.getoption.return_value = False
 
         def mock_secret_side_effect(secret_name, session):
-            if secret_name == "artifactory_servers":
-                return {}
-            elif secret_name == "default_artifactory_server":
+            if secret_name == "artifactory_servers" or secret_name == "default_artifactory_server":
```

Ruff (0.15.12)

[warning] Consider merging multiple comparisons.

Merge multiple comparisons

(PLR1714)